tools/CHANGELOG.md

```diff
@@ -105,7 +110,8 @@ Initial version, x86_64-only for now.
 - kubeadm v1.21.6
 - sonobuoy v0.53.2
 
-<!-- example comparison for future releases 
+<!-- example comparison for future releases
 [0.2.0]: https://github.com/bottlerocket-os/bottlerocket-test-system/compare/tools-v0.1.0...tools-v0.2.0 -->
+
```

Comment:
This whitespace change has me thinking we might want to bring this up to data for each release. I find these links helpful when others do them. This is not really needed in this PR but maybe we can do another one after this to add these in to historical releases.